vector/src/main/java/im/vector/app/features/home/room/list/home/invites/InvitesViewModel.kt

```diff
@@ -52,6 +65,11 @@ class InvitesViewModel @AssistedInject constructor(
 
     companion object : MavericksViewModelFactory<InvitesViewModel, InvitesViewState> by hiltMavericksViewModelFactory()
 
+    private val _invites = MutableSharedFlow<InvitesContentState>(replay = 1)
+    val invites = _invites.asSharedFlow()
+
+    var invitesCount = -1
```

Comment:
I'm wondering if this count should be part of the state, what do you think?

Reply:
this is not used by view, only inside ViewModel, so i decided to not put it to view state

Reply:
it should be declared `private` then.